webknossos/examples/image_list_to_wkw.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+
+from webknossos import Dataset
+
+INPUT_FILES = (
+    Path("path/to/file1.jpg"),
+    Path("path/to/file2.jpg"),
+)
+OUTPUT_FOLDER = Path("path/to/output_folder")
+
+
+def main() -> None:
+    """Convert a list of images to a wkw dataset."""
```

Comment:
```suggestion
    """Convert a list of images into a WEBKnossos dataset and directly add them as a new layer."""
```

Reply:
Perhaps add anoth comment about your assumptions:
- all images have the same resolution / size
- all images have the same dtype, e.g. uint8 or float
- all images are grey scale microscopy / MRI / CT scan and therefore the category is `color`
- the images are not masks/segmentations